inc/classes/settings/class-secupress-settings.php

```diff
@@ -326,6 +332,8 @@ protected function do_sections() {
 		 */
 		do_action( 'secupress.settings.before_section_' . $this->sectionnow, $with_save_button );
 
+		$this->section_is_disabled = true;
```

Comment:
did I miss something? You're setting "true" to this prop, so this is always true, right??